### PR TITLE
Small Release/README Fixes

### PR DESCRIPTION
**Pull request by holgerd77**
This PR fixes some inconsistencies within the README files discovered while browsing through the repository after having done the `beta.2` releases.

Changes are mostly header updates and headline hierarchy corrections, also some small rewrites.

The eventually somewhat unusual (or: not so commonly spotted) subtitle formatting is a table header without a body and renders like the following:

Bildschirmfoto 2020-11-12 um 10 55 51

### CODE REVIEW

**Review by evertonfraga — DISMISSED** [review]
👍 for the added consistency. 

I made two code suggestions that are not blockers, so I am approving the changes here.

File: packages/common/README.md [hunk 1]
```diff
@@ -68,9 +69,57 @@ to ease `blockNumber` based access to parameters.
...
+   the constructor or the `setChain()` method for the `chain` parameter.
+
+2. You can base your custom chain's config in a standard one, using the `Common.forCustomChain` method.
+
+## Hardforks
+
+The `hardfork` can be set in constructor like this:
+
+```typescript
+const c = new Common({ chain: 'ropsten', 'byzantium' })
```
Comment (ryanio) [inline packages/common/README.md:119]:
```suggestion
const c = new Common({ chain: 'ropsten', hardfork: 'byzantium' })
```

**Review by ryanio — APPROVED** [review]
looks great!

**Comment by ryanio** [comment]
The new GitHub feature to batch suggestions into one commit is very cool!

**Comment by holgerd77** [comment]
@ryanio oh yeah, that's indeed super-cool, otherwise having 10 commits for correcting 10 typos is purely insane. 😜  Thanks for integrating this and merging!

**Pull request merged**